TYPO3.Flow/Classes/TYPO3/Flow/Cache/Backend/SimpleFileBackend.php

```diff
@@ -396,4 +395,31 @@ public function rewind()
             $this->cacheFilesIterator->next();
         }
     }
+
+    /**
+     * @param string $cacheEntryPathAndFilename
```

Comment:
> Am 03.12.2015 um 13:26 schrieb Christian Müller <notifications@github.com>:
> 
> In TYPO3.Flow/Classes/TYPO3/Flow/Cache/Backend/SimpleFileBackend.php:
> 
> > @@ -396,4 +395,31 @@ public function rewind()
> >              $this->cacheFilesIterator->next();
> >          }
> >      }
> > +
> > +    /**
> > +     * @param string $cacheEntryPathAndFilename
> you and your return voids.... ;)
> 
> Can we remove that from the coding guideline? Or put it as CAN HAVE instead of MUST
> 

Feeding trolls here, but: No, because then you no longer know if someone has forgotten something or it was intended. 
> —
> Reply to this email directly or view it on GitHub.
>